### DISCUSSION TITLE
Events Backporting

### DISCUSSION THREAD

**Discussion by eapomazkov** [microsoft/ALAppExtensions#14188] [Q&A] [answered]
Hi!

I'm new to the repo and wanted to make sure that I understand this correctly.

New features/fixes are released only with the next major versions and some are backported to next minor, but not for the previous minor versions. Is there a way to submit a request for a feature (specifically an event) for an upcoming cumulative update of a previous minor version, or those are for hotfixes only?

Thank you!

**Comment by JesperSchulz** [comment, marked as answer]
We are not going to add event publishers or the like as hotfixes to already released minors. Generally, everything will release with the upcoming major, while some fixes will release with the upcoming minor of the current release. More here: https://github.com/Microsoft/ALAppExtensions/blob/master/FAQ.md#what-do-all-the-labels-assigned-to-issues-mean

**Reply by eapomazkov** [reply]
Got it, thanks for the quick reply!
